packages/cdktf-cli/bin/cmds/helper/init.ts

```diff
@@ -245,33 +245,31 @@ async function gatherInfo(
   projectName?: string,
   projectDescription?: string
 ): Promise<Project> {
-  if (!projectName && !projectDescription) {
-    console.log(chalkColour`\nWe will now set up the project. Please enter the details for your project.
-If you want to exit, press {magenta ^C}.
-`);
-  }
-
-  if (!projectName) {
-    // Current working directory
-    const currentDirectory = path.basename(process.cwd());
-    projectName = readlineSync.question(
-      chalkColour`{greenBright Project Name:} (default: '${currentDirectory}') `,
-      { defaultInput: currentDirectory }
-    );
-  }
-
-  if (!projectDescription) {
-    const projectDescriptionDefault =
-      "A simple getting started project for cdktf.";
-    projectDescription = readlineSync.question(
-      chalkColour`{greenBright Project Description:} (default: '${projectDescriptionDefault}') `,
-      { defaultInput: projectDescriptionDefault }
-    );
-  }
+  const currentDirectory = path.basename(process.cwd());
+  const projectDescriptionDefault =
+    "A simple getting started project for cdktf.";
+  const questions = [
```

Comment:
nit: I personally would find it more readable to create an empty array and do `if(!x) arr.push({name: "x", ...})`

Reply:
Yeah that's fair, I'll change it